README.md

```diff
@@ -17,7 +17,7 @@ Please refer to the following repositories for current Dockerfiles and documenta
     * Cx Server companion
 * [CF cli](https://github.com/SAP/devops-docker-cf-cli)
 * [Neo cli](https://github.com/SAP/devops-docker-neo-cli)
-* [MTA Builder](https://github.com/SAP/devops-docker-mta-archive-builder)
+* [Cloud MTA Builder](https://sap.github.io/cloud-mta-build-tool/)
```

Comment:
the new link points to the documentation site while the old and other tool links point to the source repositories, intended?

remove the appending slash.

Reply:
Is the documentation ready for release? 

> Creating an MTA Archive According to the MTA Deploymnet Descriptor (mtad.yaml file)
> Under construction
>

Reply:
> the new link points to the documentation site while the old and other tool links point to the source repositories, intended? Yes
> 
> remove the appending slash. done